### ISSUE TITLE
Add Issue and PR templates

### ISSUE THREAD

**Issue by kral2** [oracle-terraform-modules/terraform-oci-vcn#33]
See Github documentation:
- [x] [ issue templates](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository)
- [ ] [Pull request templates](https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)

Exact format needs to be discussed.
[terraform-provider-oci issue templates](https://github.com/terraform-providers/terraform-provider-oci/tree/master/.github/ISSUE_TEMPLATE) should be a good point to start with.

**Comment by hyder** [comment]
Yeah, we already have this in the other repos. I think we forgot to add here. We'll add them

**Comment by hyder** [comment]
Done in #36

**Comment by kral2** [comment]
do we also have (or need) a template for PR?

**Comment by hyder** [comment]
+ @Djelibeybi

**Comment by Djelibeybi** [comment]
> do we also have (or need) a template for PR?

You don't have one. You can submit one via PR by adding files to teh `.github` folder.

**Comment by kral2** [comment]
Let me rephrase my question : do we have an existing PR template on other repos that we could use as a starting point?

**Comment by Djelibeybi** [comment]
Ah, I see. No, there is no existing generic issue template.

**Comment by Djelibeybi** [comment]
This topic has started a discussion and we're probably going to implement org-wide templates in OGHO shortly. I could just copy those across to the entire `oracle-terraform-modules` org too, once they're done.

**Comment by kral2** [comment]
Thanks @Djelibeybi 
@hyder I guess we will have to write one for Pull Requests

**Comment by Djelibeybi** [comment]
We are planning to implement an org-wide `PULL_REQUEST_TEMPLATE.md` in OGHO.

**Comment by hyder** [comment]
Let's keep it open until @Djelibeybi has got the OGHO template then.

**Comment by Djelibeybi** [comment]
I would recommend we implement an org-wide `.github` repo for the community health files so that they do not have to created in each repo.

**Comment by kral2** [comment]
Having a common pattern would be best.

By org, you mean `oracle-terraform-modules` or whole OGHO?
Issue instructions can be specific to the language.

**Comment by Djelibeybi** [comment]
The highest level you can create default community health files is an organisation, which means `https://github.com/oracle-terraform-modules`. What we refer to as OGHO is `https://github.com/oracle` and it is a different organisation. These files would be created in this organisation and would only apply to the repos within it. 

The community health files have a cascading priority, from the org to the repo. If you define them at the repo level, those will override the default ones in the org. Given this organisation is subject-specific, org-wide files seem like a better option and we would probably discourage repo-specific ones.

**Comment by kral2** [comment]
perfect 👌

**Comment by kral2** [comment]
👆 a first attempt to a PR Template tailored for this Orgs repos

**Comment by Djelibeybi** [comment]
I thought we were going to do this at the org level?

**Comment by kral2** [comment]
Sorry for the confusion. My intent was to have a first version on this repo to start seeing what type of content we would need in the template, and then maybe take it to org level.

With the first attempt in PR #41, I am already calling out to actions that are specific to the way documentation is handled in this repo. Unfortunately, currently only 5-6 repos in the org share this organization.

We may need two templates : one generic, and one for the standards we would like to enforce going forward.
